erpnext/accounts/report/statement_of_account/statement_of_account.html

```diff
@@ -1,28 +1,103 @@
-<h2 class="text-center">{%= __("Statement of Account") %}</h2>
-<h4 class="text-center">
-	{% if (filters.party_name) { %}
-		{%= filters.party_name %}
-	{% } else if (filters.party && filters.party.length) { %}
-		{%= filters.party %}
-	{% } else if (filters.account) { %}
-		{%= filters.account %}
-	{% } %}
-</h4>
-
-<h6 class="text-center">
-	{% if (filters.tax_id) { %}
-	{%= __("Tax Id: ")%}	{%= filters.tax_id %}
-	{% } %}
-</h6>
-
-<h5 class="text-center">
-	{%= frappe.datetime.str_to_user(filters.from_date) %}
-	{%= __("to") %}
-	{%= frappe.datetime.str_to_user(filters.to_date) %}
-</h5>
-<hr>
+<style>
+.print-format.landscape {
+	margin: 100px;
+}
+
+.print-format td,.print-format th{
+	padding: 12px !important ;
+}
```

Comment:
@meshramsaroj Please add a space Bracket and end of style rule
Do it for all other rules